### ISSUE TITLE
Equipviewer - Black Bloxes

### ISSUE THREAD

**Issue by b2esq** [ProjectTako/ffxi-addons#21]
Running the Windower v2 (I've also tried the v1 without success).  Followed installation instructions.

Lua will load and the right size grid will appear (I can change it and I can move it), but the grid is background only without any gear.  Icons have been downloaded and unzipped and placed in the equipviewer folder.  I get no error messages at all, just no icons in the box.

At one point I tried re-downloading the core and class files and icons suddenly populated the box, but the very next game start they were gone.  I'm at a complete loss.  Any suggestions?

Thanks -- and by the way, this looks like a fantastic add on.  The videos I've seen of it online are amazing.

**Comment by ProjectTako** [comment]
b2esq,

It sounds like the icons could have incorrect pathing. It's extremely weird that it would have worked one game start, but not another though if that was the case. Can you please make sure your pathing is as follows:

addons/
    equipviewer/
        icons/
            32/
                10240.png

There should be four folders inside the icons folder, 16, 32, 48 and 64. These denote different icon sizes.

**Comment by b2esq** [comment]
Thanks for the quick response.  I’m actually traveling this weekend, so can’t yet confirm again (but I will) that I have the correct folder structure — but I do — I’ve checked multiple times (and re-download multiple times more than that, so I’ll check again).

Two questions though:

1) does it matter at all that my windower folder is “windower” and not “windower4”?  I can’t imagine it would, and everything else works, but thought I’d mention it in case the folder structures is hard baked somewhere.

2) in the v2 folder, the icon archive only has a 32 folder.  I used these, and got the 16, 48, and 64 folders from the v1.  Does it matter?  I’m using (trying to use) the 32 icons anyway, so again it shouldn’t be the issue, but thought I’d mention that as well.

Thanks,
Bill

> On Dec 12, 2019, at 11:19 PM, Project Tako <notifications@github.com> wrote:
> 

﻿
b2esq,

It sounds like the icons could have incorrect pathing. It's extremely weird that it would have worked one game start, but not another though if that was the case. Can you please make sure your pathing is as follows:

addons/
equipviewer/
icons/
32/
10240.png

There should be four folders inside the icons folder, 16, 32, 48 and 64. These denote different icon sizes.

**Comment by ProjectTako** [comment]
b2esq,

I would continue to use equipviewer v1 for now. v2 was put on here for some users that were having some lag issues with v1. I asked if it fixed it for them, and they never got back to me. v2 isn't complete of of right now.

Your windower folder name shouldn't matter. The addon uses the windower api property 'addon.path' to get the path of the addon, and then derive the icons folder from it.

Double check that you're using v1, and the icon paths are as described above.

**Comment by b2esq** [comment]
I'm going to close this.  I verified (again) that the paths/folders/contents were all as they were supposed to be, deleted them all, and started over -- all without success.  The good news is that it finally occurred to me that the only thing I'd never deleted was the config file.  I deleted that, reloaded, and moved the box into view with the command line -- and it worked.  The bad news (more odd than bad, really) is that the config file was never re-created (after several start-ups and shutdowns of both equipviewer and ffxi), and so I have to command line move the box every time -- buy hey, it works every time...

Again, thanks for the quick responses.  Love this add-on!

B2
